ml-agents/mlagents/trainers/learn.py

```diff
@@ -108,3 +108,4 @@ def main():
         p = multiprocessing.Process(target=run_training, args=(i, run_seed, options))
         jobs.append(p)
         p.start()
+        time.sleep(3)
```

Comment:
This feels like a magic number. Can it be a constant ? Also, why 3 seconds ?

Reply:
I can make it a constant. It corresponds to the time it takes between launching the binary, and the environment (with communication) displaying on the screen.

Reply:
Wanted to chime in... I agree this feels like magic and won't work in all cases.  Why not have the child process relate back when it has established communication?  e.g. with one of the classic methods: https://docs.python.org/3/library/multiprocessing.html#exchanging-objects-between-processes

Reply:
Thanks for the suggestion @harperj. I've made a new commit which uses this method, and works well.